package.json

```diff
@@ -23,6 +23,7 @@
     "react-dom": "^18.3.1",
     "react-ga4": "^2.1.0",
     "react-router-dom": "^6.22.3",
+    "react-speech-recognition": "^3.10.0",
```

Comment:
your commit still has this guy here , which i find interesting @isingoma  lets set up a meeting. today and we create a PR together